src/utils/getCalendarMonthWeeks.js

```diff
@@ -1,39 +1,39 @@
-export default function getCalendarMonthWeeks(month, enableOutsideDays) {
+import moment from 'moment';
+
+export default function getCalendarMonthWeeks(
+  month,
+  enableOutsideDays,
+  firstDayOfWeek = moment.localeData().firstDayOfWeek(),
```

Comment:
can something validate that this value is an integer 0-6, and throw a TypeError if not?